wireguard/rootfs/etc/services.d/api/run

```diff
@@ -15,6 +15,9 @@ declare public_key
 declare transfer_rx
 declare transfer_tx
 
+PORT=$(bashio::addon.port "80/tcp")
+if [[ $PORT -eq 0 ]]; then exit; fi
```

Comment:
Doesn't this get the Port that's been configured externally? i.e. the one that forwards to 80 in the container?